package.json

```diff
@@ -15,20 +14,17 @@
     "url": "https://github.com/vimaec/vim-webgl-viewer.git"
   },
   "scripts": {
-    "build": "./node_modules/.bin/gulp -- build",
-    "dist": "./node_modules/.bin/gulp -- dist",
-    "tswatch": "./node_modules/.bin/tsc.cmd -p tsconfig.json --watch",
-    "watch": "./node_modules/.bin/gulp -- watch",
-    "serve": "http-server"
+    "watch": "webpack --watch",
```

Comment:
We are missing a "distribution" step (previously "dist"). The distribution step should produce a minified (release) version of the JavaScript, ready for publishing on NPM (see: https://docs.npmjs.com/cli/v7/commands/npm-publish). We can tackle that separately.